x/foundation/client/testutil/cli_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build norace
+// +build norace
```

Comment:
Is there any reason to use this redundant constraint? Because `go:build` is enough after go 1.17.

Reply:
I know about that. However, when i save the code in the IDE, it automatically generates `+build norace`.